### ISSUE TITLE
Suggestion: Adding drops to "categories" of entities.

### ISSUE THREAD

**Issue by TeetoKaziim** [ejektaflex/Scaling-Wealth#2]
Adding scaling wealth rewards to "all hostile mobs" in a large pack involves a lot of copying and pasting. It would be pretty rad if there was a way to make an entry that applied to different broad categories of living entities. Perhaps hostile, passive, neutral, boss, perhaps others.

This is probably one of those things that's easier to say than to do. I'll understand if it's a "yeah but no".

**Comment by ejektaflex** [comment]
Yep, I've just added this to a custom build I'm using for my server. I'll push this to CurseForge once I'm sure it's bug free! It'll allow you to specify drops for an entire group of mobs, rather than copy/pasting the drops for each one. It's super simple :)

**Comment by TeetoKaziim** [comment]
I am still interested in this feature, I guess I don't know how the server test went. >.>
